patientsearch/src/js/context/UserContextProvider.js

```diff
@@ -67,28 +68,40 @@ export default function UserContextProvider({ children }) {
               encodeURIComponent(given_name)
           );
         }
-        setUser(userObj);
         // try looking up matched practitioner resource by name or email
         const allResults = await Promise.all(
           requestURLs.map((item) => fetchData(item, noCacheParam))
         ).catch(e => {
           console.log("fetch practitioner error ", e);
+          setUser(userObj);
           handleErrorCallback(e);
         });
-        if (allResults && allResults.length) {
-          const filteredResults = allResults.filter(
-              (item) => item.entry && item.entry.length > 0
-          );
-          if (filteredResults.length) {
-              practitionerId = filteredResults[0].entry[0].resource.id;
-          } else {
-            handleErrorCallback("Practitioner resource lookup failed. We are not able to find you in the system.");
-          }
+        if (!allResults || !allResults.length) {
+          setUser(userObj);
+          return;
+        }
+        const filteredResults = allResults.filter(
+            (item) => item.entry && item.entry.length > 0
+        );
+        if (filteredResults.length) {
+            practitionerId = filteredResults[0].entry[0].resource.id;
+            // NOTE - retrieving id(s) of patients whose care team the practitioner is part of
+            getPatientIdsByCareTeamParticipant(practitionerId).then(
```

Comment:
Thanks @achen2401  . As I review how this checkbox has been working, I assume that you always add the Patient.generalPractitioner to the CareTeam - is that the case?

Reply:
@mcjustin yes, it is the case, see [here](https://github.com/uwcirg/isacc-messaging-client-sof/blob/d1b1057e9cda06fec0f091a172b0b3d16ca1d905/src/components/Summary.tsx#L999)

Reply:
Yes @mcjustin , the generalPractioner is always in the CareTeam.  (I know as Amy and I chatted about the same thing just days ago).